helpers/scraper.js

```diff
@@ -0,0 +1,366 @@
+            names.push(data);
+            let splitData = data[0].split("_");
+            //console.log(data);
+            if (splitData[splitData.length - 1] == 'State') {
+              splitData[splitData.length - 1] = 'St';
+              names.push(splitData.join('_'));
+            }
+            if (data[0].split(' ').length > 1 && notTitle(data[0])) {
+              count++;
+            }
+          });
+          resolve(names);
+        } else {
+          console.log(resp.statusCode);
+          reject(err);
+        }
+      }
+    )});
+}
+
+  //console.log(teams);
+  // if you print teams, you'll see that it's empty.
+const parseDate = function(date) {
+  return new Date(date);
+}
+
+const earliestDate = parseDate('09/08/2017');
+
+const delay = function(ms) {
+  return new Promise(resolve => setTimeout(resolve, ms));
+}
+
+const sleep = async () => {
+  console.log('Taking a break...');
+  await delay(((Math.random() * 5) + 1) * 1000)
+  console.log('Two second later');
+}
+
+const formatDate = function(date) {
+  if (date == "#") {
+    return "01/01/2000";
+  }
+  let parts = date.split('/');
+  // console.log("****************************");
+  // console.log(parts[2]);
+  parts[2] = '20'.concat(parts[2]).replace('\n', '');
+  let result = parts.join('/');
+  // console.log('---------------------------');
+  // console.log(result);
+  // console.log(parts);
+  // console.log('------------------------------')
+  // console.log("*******************************")
+  return result;
+}
+
+const fetchMeets = async (url) => {
+  return new Promise((resolve, reject) => {
+        let meets = [];
+        request(url, function(err, resp, body) {
+        if (!err && resp.statusCode == 200) {
+            let $ = cheerio.load(body);
+            $('tr', '.data, .scroll').each(function() {
+              if (!$(this).find('.date').text()) {
+                //console.log("No valid date for this meet: ");
+                //console.log($(this).find('a').attr('href'))
+              } 
+              else {
+                let date = parseDate(formatDate($(this).find('.date').text()));
+                //console.log(date.getTime());
+                //console.log(date);
+                //console.log('-----------------');
+                //console.log(earliestDate.getTime());
+
+                if (date.getTime() >= earliestDate.getTime()) {
+                  //console.log($(this).find('.date').text());
+                  //console.log($(this).find('a').attr('href'));
+                  let meetLink = $(this).find('a').attr('href').replace('//', '');
+                  if (meetLink.includes('www')){
+                      meets.push('https://'.concat(meetLink));
+                      //console.log(meets);
+                  }
+                }
+              }
+            });
+        }
+        //console.log(meets);
+        //console.log(url);
+        //console.log(meets);
+        resolve(meets);
```

Comment:
Probably should remove all of these commented console statements

Reply:
yeah i'll do that